Velodrome tokenlist

stas opened the pull request:
Added some more tokens from our token list. Removed the ones we don't have data for.

Added an extra TODO to the list of things when submitting the PR. Eg.:

Pricing checks:
* [x] `coin_id` represents the ID of the coin on coinpaprika.com
* [x] all the coins are active on coinpaprika.com (please remove inactive ones)

There's a bunch of tokens without pricing data, should I comment these and add them to the PR or leave it like this?
```
    -- TODO: Update later...
    -- ("NULL","optimism","sLINK","0xc5db22719a06418028a40a9b5e9a7c02959d0d08",18),
    -- ("NULL","optimism","sBTC","0x298b9b95708152ff6968aafd889c6586e9169f1d",18),
    -- ("NULL","optimism","USX","0xbfd291da8a403daaf7e5e9dc1ec0aceacd4848b9",18),
    -- ("NULL","optimism","AELIN","0x61baadcf22d2565b0f471b291c475db5555e0b76",18),
    -- ("NULL","optimism","ZIP","0xfa436399d0458dbe8ab890c3441256e3e09022a8",18),
    -- ("NULL","optimism","sSOL","0x8b2f7ae8ca8ee8428b6d76de88326bb413db2766",18),
    -- ("NULL","optimism","sAVAX","0xb2b42b231c68cbb0b4bf2ffebf57782fd97d3da4",18),
    -- ("NULL","optimism","sMATIC","0x81ddfac111913d3d5218dea999216323b7cd6356",18),
    -- ("NULL","optimism","sAAVE","0x00b8d5a5e1ac97cb4341c4bc4367443c8776e8d9",18),
    -- ("NULL","optimism","sUNI","0xf5a6115aa582fd1beea22bc93b7dc7a785f60d03",18),
    -- ("NULL","optimism","MAI","0xdfa46478f9e5ea86d57387849598dbfb2e964b02",18),
    -- ("NULL","optimism","QI","0x3f56e0c36d275367b8c502090edf38289b3dea0d",18),
    -- ("NULL","optimism","VELO","0x3c8b650257cfb5f272f799f5e2b4e65093a11a05",18),
    -- ("NULL","optimism","DOLA","0x8ae125e8653821e851f12a49f7765db9a9ce7384",18),
    -- (NULL, "optimism", "aCRV", "0x764ad60e1b81f6cacfec1a2926393d688d4493e6", 18),
    -- (NULL, "optimism", "BAXA", "0xdb9888b842408b0b8efa1f5477bd9f351754999e", 18),
    -- (NULL, "optimism", "jEUR", "0x79af5dd14e855823fa3e9ecacdf001d99647d043", 18),
    -- (NULL, "optimism", "LPF", "0x0b3e851cf6508a16266bc68a651ea68b31ef673b", 18),
    -- (NULL, "optimism", "OPENX", "0xc3864f98f2a61a7caeb95b039d031b4e2f55e0e9", 18),
    -- (NULL, "optimism", "OPL", "0xa8d83cd5102b0a412931b12255d218b0b0e7152c", 18),
    -- (NULL, "optimism", "OPP", "0x676f784d19c7f1ac6c6beaeaac78b02a73427852", 18),
    -- (NULL, "optimism", "pETHo", "0x6ca558bd3eab53da1b25ab97916dd14bf6cfee4e", 18),
    -- (NULL, "optimism", "RFWSTETH", "0xeaeadac73baaf4cb8b024de9d65b2eefa722856c", 18),
    -- (NULL, "optimism", "sETHo", "0x09448876068907827ec15f49a8f1a58c70b04d45", 18),
    -- (NULL, "optimism", "LIZ", "0x3bb4445d30ac020a84c1b5a8a2c6248ebc9779d0", 18),
    -- (NULL, "optimism", "MASK", "0x3390108e913824b8ead638444cc52b9abdf63798", 18),
    -- (NULL, "optimism", "O3", "0xee9801669c6138e84bd50deb500827b776777d28", 18),
    -- (NULL, "optimism", "SONNE", "0x1db2466d9f5e10d7090e7152b68d62703a2245f0", 18),
    -- (NULL, "optimism", "TheDAO", "0xd8f365c2c85648f9b89d9f1bf72c0ae4b1c36cfd", 16),
    -- (NULL, "optimism", "USD+", "0x73cb180bf0521828d8849bc8cf2b920918e23032", 6),
    -- (NULL, "optimism", "USX", "0xbfd291da8a403daaf7e5e9dc1ec0aceacd4848b9", 18),
    -- (NULL, "optimism", "xOPENX", "0x5ae99cce8b5bc9c639a27ccc2c84448c7a6a0a12", 18),
    -- (NULL, "optimism", "xOpenX", "0x2513486f18eee1498d7b6281f668b955181dd0d9", 18),
```

dune-eng commented:
Workflow run id 3267287117 approved.

dune-eng commented:
Workflow run id 3267298022 approved.

dune-eng commented:
Workflow run id 3267827534 approved.

augustog reviewed (changes requested):
Sorry for the slow response here!

I checked the list of tokens and seems two more have inactive feeds on coinpaprika. I commented them inline. The rest looks good to go.

As for adding inactive / unavailable feeds as commented-out lines, we've been doing it with others so I don't think it should be an issue. We should probably have a more structured approach but for now this will do it.

Oh and thanks for extending the checklist with relevant tasks for these PRs!

augustog commented on models/prices/prices_tokens.sql:
```diff
@@ -1114,9 +1115,19 @@ VALUES
-    
-
+    ("0xbtc-0xbitcoin", "optimism", "0xBTC", "0xe0bb0d3de8c10976511e5030ca403dbf4c25165b", 8),
+    ("bitbtc-bitbtc", "optimism", "BitBTC", "0xc98b98d17435aa00830c87ea02474c5007e1f272", 18),
```
inactive on cp :(

augustog commented on models/prices/prices_tokens.sql:
```diff
@@ -1114,9 +1115,19 @@ VALUES
+    ("eqz-equalizer", "optimism", "EQZ", "0x81ab7e0d570b01411fcc4afd3d50ec8c241cb74b", 18),
+    ("gysr-geyser", "optimism", "GYSR", "0x117cfd9060525452db4a34d51c0b3b7599087f05", 18),
+    ("han-hanchain", "optimism", "HAN", "0x50bce64397c75488465253c0a034b8097fea6578", 18),
+    ("hnd-hundred-finance", "optimism", "HND", "0x10010078a54396f62c96df8532dc2b4847d47ed3", 18),
```
inactive on cp :(

augustog commented:
And thanks for the addition to the checklist!! :)

augustog commented:
As for adding them and keeping commented out, I think that's good for now, we've been doing it for a few others.

dune-eng commented:
Workflow run id 3322078102 approved.

dune-eng commented:
Workflow run id 3322098494 approved.

dune-eng commented:
Workflow run id 3322098505 approved.

jeff-dude reviewed (changes requested):
please merge dune main branch into this branch, there were tests added to prices file and i believe your entries may fail on duplicates

i tried but don't have access

dune-eng commented:
Workflow run id 3337618820 approved.

dune-eng commented:
Workflow run id 3337761588 approved.

dune-eng commented:
Workflow run id 3337791515 approved.

dune-eng commented:
Workflow run id 3337791509 approved.

stas commented:
Removed dups/wrong ids, added `BANK`. 

@jeff-dude any chance the tests could actually list the records with issues, or alternatively force records to be sorted alphabetically? :pray: 

Thanks for reviewing this!

jeff-dude commented:
> Removed dups/wrong ids, added `BANK`.
> 
> @jeff-dude any chance the tests could actually list the records with issues, or alternatively force records to be sorted alphabetically? 🙏
> 
> Thanks for reviewing this!

we're considering looking into spitting out the dbt results from the gh action into PRs, but it's not actively being worked on. the short term approach is to compile your code, go to target folder and find the test, run query on dune for review (especially if failing)
